### PR TITLE
All readme.md files have been translated

### PR DESCRIPTION
**Pull request by emreyunusbas**
Necessary address updates were made

### CODE REVIEW

**Review by YasinDehfuli — APPROVED** [review]

**Review by YasinDehfuli — CHANGES_REQUESTED** [review]
hi, @emreyunusbas, because of some conflicts i cant accept and merge your changes. please fix the reviewed comments.  after conflict bug fix i can accept your Turkey translate & add you to our contributers

File: assets/steps-tr/README.yolo-tr.md
[the remark this answers is not in the document]
Comment (emreyunusbas) [reply assets/steps-tr/README.yolo-tr.md:1]: I think I have corrected the existing conflicts, you can check and come back or add it directly if it is correct. and I have corrected the translations in the course and other files in yolo

[the remark this answers is not in the document]
Comment (YasinDehfuli) [reply assets/steps-tr/README.yolo-tr.md:1]: image

as you can see you need to remove turkish button in README.md to fix the conflict. 
i presure your job, and i want to accept it but we need to fix conflict in README.md

**Comment by YasinDehfuli** [comment]
Please Remove main README.md file i will add button on turkey

**Pull request closed without merging**